x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/datastreams/DataStreamFeatureSetUsage.java

```diff
@@ -49,6 +50,12 @@ protected void innerXContent(XContentBuilder builder, Params params) throws IOEx
         super.innerXContent(builder, params);
         builder.field("data_streams", streamStats.totalDataStreamCount);
         builder.field("indices_count", streamStats.indicesBehindDataStream);
+        if (DataStream.isFailureStoreFeatureFlagEnabled()) {
+            builder.startObject("failure_store");
+            builder.field("enabled", streamStats.failureStoreEnabledDataStreamCount);
```

Comment:
Small bike shed (a.k.a. dog house?): I find it ever so slightly confusing that `data_streams.enabled` is a boolean but `data_streams.failure_store.enabled` is a number. What about doing `enabled_count`? I don't feel too strongly about this, so it wouldn't be a blocker for me.

Reply:
+1